packages/camera/camera/example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.2'
```

Comment:
I understand if you want this in another pr but we also need to migrate to the plugin version of this import.

Reply:
Yes we should do that at some point, but would prefer for it not to be this pr